Gruntfile.js

```diff
@@ -44,18 +44,30 @@ module.exports = function(grunt) {
       },
       start_end_options: {
         options: {
-          start_comment: '{test}',
-          end_comment: '{/test}',
+          interceptionCheck: true,
+          blocks: [
+            {
+              start_block: '/* {test} */',
+              end_block: '/* {/test} */',
+            },
+            {
+              start_block: '/* test-code */',
+              end_block: '/* end-test-code */',
+            },
+          ]
         },
         src: 'tmp/start_end_options.js',
       },
       pattern_options: {
         options: {
-          pattern: / *console\.log\(['"a-z]+\)\n?/g
+          patterns: / *console\.log\(['"a-z]+\)\n?/g
```

Comment:
Yes, your variant is better. Will take case of that in the evening 

// I do not use this option in my project, so I just went with existing code..